themes-default/slim/views/config_backuprestore.mako

```diff
@@ -113,7 +98,8 @@ const startVue = () => {
                         <div class="field-pair">
                             Select the backup file you wish to restore:
                             <br><br>
-                            <input type="text" name="backupFile" id="backupFile" class="form-control input-sm input350"/>
+                            <!-- <input v-model="restore.file" type="text" name="backupFile" id="backupFile" class="form-control input-sm input350"/> -->
+                            <file-browser name="backupFile" ref="backupFileBrowser" id="backupFile" title="Select Show Location" :initial-dir="restore.file" :include-files="true" @update:location="restore.file = $event"></file-browser>
```

Comment:
1. I don't remember why I added the `ref=` in editShow but it's not needed.
`id=` is not needed here either.
2. `initial-dir` is not needed in this case, as it will always be empty (no config variable.
3. You need to add `local-storage-key="backupFile"`
4. As explained in the toggle button PR, boolean properties can be `<file-browser ... include-files>`.

Reply:
4: didnt know tnx